pkg/result/processors/autogenerated_exclude_test.go

```diff
@@ -92,3 +103,11 @@ and this text also`,
 		assert.Equal(t, tc.doc, doc)
 	}
 }
+
+// Issue 954: Some lines can be very long, e.g. auto-generated
+// embedded resources. Reported on file of 86.2KB.
+func TestGetDocFileWithLongLine(t *testing.T) {
```

Comment:
This change replaces the whole implementation of `getDoc()`, so I also added a test for confirming previous issue (#954).